test/integration2/query/select/where.spec.js

```diff
@@ -632,7 +639,12 @@ describe('Where', function () {
 
         it("doesn't find data using whereLike when different case sensitivity", async () => {
           const result = await knex('accounts').whereLike('email', 'Test1%');
-          expect(result).to.deep.equal([]);
+          if (isSQLite(knex)) {
```

Comment:
please add comment that sqlite only supports case-insensitive search

Reply:
Done